blockchain.com/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@chainlink/blockchain.com",
+  "name": "@chainlink/blockchain.com-adapter",
```

Comment:
We should probably depreciate already published packages, maybe [using this tool to make it easy](https://stackoverflow.com/a/56197360). We can maybe also reset the version to `0.0.1`, what do you think?

Reply:
I'm not sure if it's necessary to do it, as only a handful of adapters have been published so far, a few of them were published with "-adapter" already appended, and afaik, only we are using these packages. I don't think any of the versions have ever been upgraded either.

If you're aware of any packages/adapters that have been published with a version > 0.0.1, we can probably update them.

Reply:
Yeah, I think that PoR adapters are on `0.0.3` as we messed up the TS builds a few times and published the package without types.

Reply:
Thanks for adding the suggestions, really helpful!